lod_api/tools/helper.py

```diff
@@ -39,12 +39,15 @@ def call(es, action, index, **kwargs):
         """ Call a method of the elasticsearch api on a specified index
         with multiple variable kwargs as options to each call. """
         server_version = int(es.info()['version']['number'][0])
-        if server_version < 7:
-            return getattr(es, action)(index=index, **kwargs)
-        elif server_version >= 7:
-            # ignore doc_type keyword argument
-            kwargs.pop("doc_type")
-            return getattr(es, action)(index=index, **kwargs)
+        if server_version >= 7:
+            if '_source_exclude' in kwargs:
+                kwargs['_source_excludes'] = kwargs.pop('_source_exclude')
```

Comment:
Ich würde das anders herum machen. Also:
```python
if server_version < 7:
    if '_source_excludes' in kwargs:
        kwargs['_source_exclude'] = kwargs.pop('_source_excludes')
```
Damit bleiben wir mit dem ES-Wrapper möglichst nah an der aktuellen Version der elasticsearch-API. Allerdings muss man damit die Aufrufe vom ES-Wrapper in den anderen Codestellen noch ändern.